pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -302,14 +302,9 @@ func (exp *flowExporter) sendFlowRecords() error {
 
 			if flowexporter.IsConnectionDying(&record.Conn) {
 				// If the connection is in dying state or connection is not in conntrack table,
-				// we will delete the flow records from records map.
+				// we set the DoneExport flag to do the deletion later.
 				klog.V(2).Infof("Deleting the inactive flow records with key: %v from record map", key)
-				if err := exp.flowRecords.DeleteFlowRecordWithoutLock(key); err != nil {
```

Comment:
I think this function `DeleteFlowRecordWithoutLock(key)` can be removed, right?

Reply:
Yes. Done.